### ISSUE TITLE
Crash when loading Ballgame then Country Roads from c1-msic-net

### ISSUE THREAD

**Issue by dcwatson** [avaraline/Avara#168]
```
INFO: LOADING LEVEL country-roads.alf FROM c1-misc-net
free(): invalid size
Process 374237 stopped
* thread #1, name = 'Avara', stop reason = signal SIGABRT
    frame #0: 0x00007ffff77d4d22 libc.so.6`raise + 322
libc.so.6`raise:
->  0x7ffff77d4d22 <+322>: movq   0x108(%rsp), %rax
    0x7ffff77d4d2a <+330>: subq   %fs:0x28, %rax
    0x7ffff77d4d33 <+339>: jne    0x7ffff77d4d54            ; <+372>
    0x7ffff77d4d35 <+341>: movl   %r8d, %eax
(lldb) bt
* thread #1, name = 'Avara', stop reason = signal SIGABRT
  * frame #0: 0x00007ffff77d4d22 libc.so.6`raise + 322
    frame #1: 0x00007ffff77be862 libc.so.6`abort + 278
    frame #2: 0x00007ffff7816d28 libc.so.6`__libc_message + 616
    frame #3: 0x00007ffff781e92a libc.so.6`malloc_printerr + 26
    frame #4: 0x00007ffff781fd0c libc.so.6`_int_free + 796
    frame #5: 0x00007ffff78239e8 libc.so.6`__libc_free + 104
    frame #6: 0x00005555556fe570 Avara`DisposePtr(p="plier") at Memory.cpp:15:9
    frame #7: 0x00005555556dfe77 Avara`DeallocParser() at Parser.cpp:1099:9
    frame #8: 0x00005555556e0476 Avara`InitParser() at Parser.cpp:1223:9
    frame #9: 0x00005555556e07b4 Avara`LoadALF(levelName=error: summary string parsing error) at LevelLoader.cpp:284:5
    frame #10: 0x00005555556c70ef Avara`CAvaraAppImpl::LoadLevel(this=0x0000555555a124c0, set=error: summary string parsing error, levelTag=error: summary string parsing error, sendingPlayer=0x000055555622f820) at CAvaraApp.cpp:238:8
    frame #11: 0x00005555556a559c Avara`CNetManager::ReceiveLoadLevel(this=0x0000555556239cc0, senderSlot=0, theSetAndTag="c1-misc-net/country-roads.alf", seed=12) at CNetManager.cpp:456:24
    frame #12: 0x0000555555709c57 Avara`CProtoControl::DelayedPacketHandler(this=0x0000555556233840, thePacket=0x00005555565e00a4) at CProtoControl.cpp:95:21
    frame #13: 0x0000555555709759 Avara`DelayedProtoHandler(thePacket=0x00005555565e00a4, userData="P") at CProtoControl.cpp:37:24
    frame #14: 0x0000555555709554 Avara`CCommManager::ProcessQueue(this=0x00005555559aec80) at CCommManager.cpp:363:33
    frame #15: 0x000055555570ad05 Avara`CUDPComm::ProcessQueue(this=0x00005555559aec80) at CUDPComm.cpp:348:19
    frame #16: 0x00005555556a43e1 Avara`CNetManager::ProcessQueue(this=0x0000555556239cc0) at CNetManager.cpp:199:21
    frame #17: 0x0000555555685b60 Avara`CAvaraGame::GameTick(this=0x0000555556238620) at CAvaraGame.cpp:832:13
    frame #18: 0x00005555556c5cb9 Avara`CAvaraAppImpl::idle(this=0x0000555555a124c0) at CAvaraApp.cpp:118:17
    frame #19: 0x00005555557675c6 Avara`nanogui::mainloop(refresh=16) at common.cpp:82:25
    frame #20: 0x00005555557bcf3f Avara`main(argc=1, argv=0x00007fffffffde78) at Avara.cpp:88:5
    frame #21: 0x00007ffff77bfb25 libc.so.6`__libc_start_main + 213
    frame #22: 0x00005555555f234e Avara`_start + 46
```

**Comment by rherriman** [comment]
This ... appears to be fixed? Maybe as of #217. Can anybody else verify?

**Comment by assertivist** [comment]
I'm not able to reproduce this on main locally either, closing.
